modules/assisted-installer-completing-the-installation.adoc

```diff
@@ -0,0 +1,49 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="completing-the-installation_{context}"]
+= Completing the installation
+
+Once the cluster is installed and initialized, the {ai-full} will indicate that installation is complete. You will have access to the console URL, the `kubeadmin` username and password, and the `kubeconfig` file. Additionally, the {ai-full} will provide cluster details including the {product-title} version, base domain, CPU architecture, API and Ingress IP addresses, and the cluster and service network IP addresses.
+
+.Procedure
+
+. Install the `oc` CLI tool if necessary.
+
+. Make a copy of the `kubeadmin` username and password.
+
+. Download the `kubeconfig` file and copy it to the `auth` directory under your working directory:
```

Comment:
Dropping a note somewhere that it's available to download for 24 hours after installation is complete